csrf/app/csrf.go

```diff
@@ -152,7 +153,7 @@ func getFullRequestURL(c *revel.Controller) (requestUrl *url.URL) {
 		} else {
 			requestUrl.Scheme = "http"
 		}
-		fixedUrl := requestUrl.Scheme + "://" + c.Request.Host + c.Request.URL.RawPath
+		fixedUrl := requestUrl.Scheme + "://" + c.Request.Host + c.Request.GetPath()
```

Comment:
https://github.com/revel/revel/blob/a3d7a7c23ca885cc5036d3641ede49ce4a14ee2e/http.go#L120

This should work.

Reply:
@ptman `GetPath` returns `URL.Path` object so the following should work equivalently and be slightly more optimal. 

		fixedUrl := requestUrl.Scheme + "://" + c.Request.Host + c.Request.URL.Path

Reply:
sure